org.jacoco.core.test/src/org/jacoco/core/internal/analysis/filter/RecordsFilterTest.java

```diff
@@ -143,6 +143,32 @@ public void should_not_filter_custom_equals_method() {
 		assertIgnored();
 	}
 
+	@Test
+	public void should_not_filter_field_int() {
```

Comment:
Why should the getter *not* be filtered here? Looks like the test setup is wrong. I would recommend to start with postive test cases like `should_filter_int_accessor` etc.

Afterwards add negative tests e.g. `should_not_filter_accessor_when_name_does_not_match`

Reply:
Ohhh you're right, I got the method name wrong. It's actually a "should filter" test. I'll correct

Reply:
In this case you should call assertMethodIgnored(m);

Reply:
Fixed!